securitas/tests/unit/controller/test_registration.py

```diff
@@ -9,12 +9,6 @@
 from securitas.utility.defaults import DEFAULTS
 
 
-@pytest.fixture
-def cleanup_dummy_user():
-    yield
-    ipa_admin.user_del('dummy')
```

Comment:
I'm not against moving it to the more generic `conftest.py` but you don't seem to be using that fixture for your tests. I've put it here because I didn't think any test would need it besides the registration tests (but happy to be proven wrong).

Reply:
Yeah - I was going to use it but realised I didn't need it.  I think its more that it seems strange to me to have any fixtures in any test file. My real preference would be `registration_fixtures.py` etc but I know scoping here makes that redundant. Happy to revert if you would prefer.

Reply:
I'm just trying to avoid the conftest.py file growing without limits because of fixtures that are only used in one place. Yeah if you don't use it please revert this change.